packages/common/src/stores/AccountController.ts

```diff
@@ -196,9 +196,6 @@ export default class AccountController {
 
     if (response) {
       await this.afterLogin(response.user, response.customerConsents, accessModel);
-
-      await this.favoritesController?.restoreFavorites();
-      await this.watchHistoryController?.restoreWatchHistory();
```

Comment:
What about: 

```tsx
await Promise.allSettled([
  this.favoritesController?.restoreFavorites(),
  this.watchHistoryController?.restoreWatchHistory(),
]);
```

This will swallow the error and should fix the login error.

Reply:
A better version of my suggestion, let's do this (and accept the side effect @langemike described).

Reply:
On second thought, it might be bad practice because we will never see this error again...

Perhaps this is better:

```tsx
try {
  await this.favoritesController?.restoreFavorites();
  await this.watchHistoryController?.restoreWatchHistory();
} catch (error: unknown) {
  logDev('Failed to restore favorites or watch history', error);
}
```

The downside of this is that if the favorites call fails, the restore watch history function is never called. We could do this, but, personally, I don't like it that much:

```tsx
  await this.favoritesController?.restoreFavorites().catch(logDev);
  await this.watchHistoryController?.restoreWatchHistory().catch(logDev);
```

Reply:
```javascript
await Promise.allSettled([
  this.favoritesController?.restoreFavorites(),
  this.watchHistoryController?.restoreWatchHistory()
]).then((results) => {
  results.forEach((result, index) => {
    if (result.status === 'rejected') {
      logDev('Failed to restore favorites or watch history', result.reson);
    }
  });
});
```

Reply:
Thanks for your suggestions. I prefer using the following approach, because it keeps the code simple/clean.
```js
 await this.favoritesController?.restoreFavorites().catch(logDev);
 await this.watchHistoryController?.restoreWatchHistory().catch(logDev);
 ```
 
 I hope you agree.

Reply:
Of course!